deploy/examples/create-external-cluster-resources.py

```diff
@@ -1807,6 +1817,19 @@ def upgrade_user_permissions(self, user):
             print(f"user {user} not found for upgrading.")
             return
         existing_caps = json_out[0]["caps"]
+        if user == "client.healthchecker":
+            # when admin has not provided rgw pool name during upgrade,
```

Comment:
As we are not documenting it in the above --upgrade flag help,

So we should not document it.

Reply:
I think it helps in understanding the code what we are doing, so keeping it here also